seeders/03_drivers.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+    up: queryInterface => {
+        const arr = [
+            {
+                id: 1,
+                first_name: 'Jhon',
+                last_name: 'Wick',
+                email: 'wick@jhon.is',
+                mobile_no: '8585959524',
+                cab_type_id: 1,
+                cab_number: 'MH 20 9211',
+                gender: 'male'
+            },
+            {
+                id: 2,
```

Comment:
haha, nice to use pop culture names here, but I would love if you found your way around using faker js

https://github.com/faker-js/faker

for seeding the database, let me know if you are facing problems in this

Reply:
Ok, I will try to use faker for seeding pourpose.